pointers-and-errors.md

```diff
@@ -243,7 +243,7 @@ func (b Bitcoin) String() string {
 }
 ```
 
-As you can see, the syntax for creating a method on a type alias is the same as it is on a struct.
+As you can see, the syntax for creating a method on a type decleration is the same as it is on a struct.
```

Comment:
Good spot, but it's spelt `declaration`. If you could make the edit I'll merge it. Thanks.

Reply:
oops 😅 , fixed. thanks 😄